src/modules/auth/index.js

```diff
@@ -56,6 +58,18 @@ export default class SeaCatAuthModule extends Module {
 			name: 'Resource detail',
 			component: ResourcesDetailContainer
 		});
+		app.Router.addRoute({
+			path: '/auth/deletedresources',
```

Comment:
@petrKavulok This path is incorrect, it should be under the `/auth/resources/deleted`

Reply:
Eventually it can be also `/auth/resources/!deleted`